.nvmrc

```diff
@@ -0,0 +1 @@
+18.12.1
```

Comment:
Was this intended to be commited? If so, what is this required by (Prettier, GH Workflow)?

Reply:
I've added some docs regarding this in https://github.com/ubccsss/ubccsss.org/pull/348/commits/6897aede1f15120289a8d863c52750f8194687fc. Let me know if it looks good. @kewbish

Reply:
This makes sense, looks great! Thank you.